tests/common/plugins/sanity_check/recover.py

```diff
@@ -103,13 +105,27 @@ def _neighbor_vm_recover_bgpd(node=None, results=None):
     results[nbr_host.hostname] = result
 
 
-def neighbor_vm_restore(duthost, nbrhosts, tbinfo):
+def _neighbor_vm_recover_config(node=None, results=None):
+    if isinstance(node["host"], SonicHost):
+        config_reload(node["host"])
+    return results
+
+
+def neighbor_vm_restore(duthost, nbrhosts, tbinfo, result):
```

Comment:
The new parameter "result" will cause an error in fixture advanceboot_neighbor_restore(https://github.com/sonic-net/sonic-mgmt/blob/a45d53f195f0bfe3fd0edcfdbc3f1e998b246681/tests/platform_tests/conftest.py#L564)
neighbor_vm_restore is called in the fixture with still 3 parameters, @Pterosaur could you take a look? Thanks.

Reply:
I've opened a issue for this: https://github.com/sonic-net/sonic-mgmt/issues/7582